packages/plugin-core/src/commands/ShowPreview.ts

```diff
@@ -301,6 +304,23 @@ export class ShowPreviewCommand extends BasicCommand<
   constructor(previewPanel: vscode.WebviewPanel) {
     super();
     this._panel = previewPanel;
+
+    if (UI_SHOW_PREVIEW_CMD === undefined) {
+      UI_SHOW_PREVIEW_CMD = this;
+    }
+  }
+
+  async syncEditorToPreviewPlacement(
```

Comment:
nit: should this one actually be named 'sync-preview-to-editor'? As in - we're updating the position of the preview to the current position in the editor